src/main/java/de/retest/recheck/review/ignore/PixelDiffFilter.java

```diff
@@ -25,6 +29,8 @@
 	private final boolean specifiedAsDouble;
 	private final double pixelDiff;
 
+	private static final Set<String> ignoredKeys = new HashSet<>( Collections.singletonList( "style" ) );
```

Comment:
Should be moved above the instance fields, where also the other `static` field is.

Reply:
For now?
```suggestion
	private static final Set<String> ignoredKeys = Collections.singleton( "style" );
```